superset/dashboards/filters.py

```diff
@@ -23,6 +24,19 @@
 from superset.views.base import BaseFilter, get_user_roles
 
 
+class DashboardTitleOrSlugFilter(BaseFilter):  # pylint: disable=too-few-public-methods
+    name = _("Title or Slug")
+    arg_name = "title_or_slug"
+
+    def apply(self, query, value):
+        return query.filter(
+            or_(
+                Dashboard.dashboard_title.ilike(value + "%"),
```

Comment:
Should there be `%` at the beginning of these filters as well?

Reply:
Good idea, changed it